src/dags/basiskaart.py

```diff
@@ -155,84 +147,84 @@ def create_basiskaart_dag(is_first: bool, table_name: str, select_statement: str
             },
             dag=dag,
         )
+        for table_name, select_statement in TABLES_TO_CREATE.items()
+    ]
 
-        # 4. Check for changes in temp table to merge in target table
-        change_data_capture = PgComparatorCDCOperator(
-            task_id="change_data_capture",
+    join_2 = join(2)
+
+    change_data_capture = [
+        PgComparatorCDCOperator(
+            task_id=f"change_data_capture_for_{table_name}",
             source_table=f"{DAG_ID}_{table_name}_temp",
             target_table=f"{DAG_ID}_{table_name}",
         )
+        for table_name in TABLES_TO_CREATE.keys()
+    ]
+
+    join_3 = join(3)
 
-        # 5. Create mviews for T-REX tile server
-        create_mviews = PostgresOperator(
-            task_id="create_mviews",
+    # Create mviews for T-REX tile server
+    create_mviews = [
+        PostgresOperator(
+            task_id=f"create_mviews_for_{table_name}",
             sql=CREATE_MVIEWS,
             params={"base_table": table_name, "dag_id": DAG_ID},
         )
+        for table_name in TABLES_TO_CREATE.keys()
+    ]
+
+    provenance_translation = ProvenanceRenameOperator(
+        task_id="rename_columns",
+        dataset_name=DAG_ID,
+        prefix_table_name=f"{DAG_ID}_",
+        rename_indexes=False,
+        pg_schema="public",
+    )
 
-        # 6. Rename COLUMNS based on Provenance
-        provenance_translation = ProvenanceRenameOperator(
-            task_id="rename_columns",
-            dataset_name=DAG_ID,
-            prefix_table_name=f"{DAG_ID}_",
-            rename_indexes=False,
-            pg_schema="public",
-        )
-
-        # 7. Drop temp table
-        clean_up = PostgresOperator(
-            task_id="drop_temp_table",
+    clean_up = [
+        PostgresOperator(
+            task_id=f"drop_temp_table_{table_name}",
             sql=[
                 f"DROP TABLE IF EXISTS {DAG_ID}_{table_name}_temp CASCADE",
             ],
         )
+        for table_name in TABLES_TO_CREATE.keys()
+    ]
 
-        # 8. Trigger next DAG to run (estafette)
-        trigger_next_dag = TriggerDynamicDagRunOperator(
-            task_id="trigger_next_dag",
-            dag_id_prefix=f"{DAG_ID}_",
-            trigger_rule="all_done",
-        )
-
-        # 9. Grant database permissions
-        grant_db_permissions = PostgresPermissionsOperator(task_id="grants", dag_name=DAG_ID)
+    grant_db_permissions = PostgresPermissionsOperator(task_id="grants", dag_name=DAG_ID)
```

Comment:
The `TriggerDynamicDagRunOperator` has been dropped now. However, that was meant to serialize the DAGs, to avoid some "thundering herd" of DAGs. According to the main comment of this commit, everthing has been parallelized now. So, be carefull to check performance when this code goes to production.

Reply:
Ah, I see. I was initially in doubt how to deal with `TriggerDynamicDagRunOperator`. Thing is that `SqlAlchemyCreateObjectOperator` creates the tables for all dynamic DAGs in one go. Felt a bit weird to have that executed in each dynamic DAG. Executing it only once, namely in the first dynamic DAG being run, felt awkward too, as I feel that each DAG should list all the tasks in needs to achieve its goal. As such creating a dependency between DAG didn't feel right

Will keep an eye on the performance. Especially now that I have seen this setting in `airflow.cfg`:
```
# The amount of parallelism as a setting to the executor. This defines
# the max number of task instances that should run simultaneously
# on this airflow installation
parallelism = 32
```

Reply:
You got me sufficiently worried that I have tried to address the problem by creating a subdag per table. This now (still needs to be commited) looks like:
![20210713_155316_screenshot](https://user-images.githubusercontent.com/4290278/125464445-84f06d0a-8101-440d-86af-c2c36740c9eb.png)

Reply:
`SubDag`'s are a pain. So that solution didn't work. I have now serialized all tasks and used `TaskGroup`s to visually partition the sub units of work in the GUI. This is how it looks;
![20210713_171835_screenshot](https://user-images.githubusercontent.com/4290278/125481448-752937cf-a481-4f63-96b3-095c415a83be.png)

It should prevent the thundering herd problem (overwhelming the DB).